packages/components/src/components/data-table/_data-table-inline-edit-cell.scss

```diff
@@ -299,8 +299,8 @@
     padding: 1rem;
     color: $text-error;
     font-size: rem(12px);
-    background-color: $inverse-01;
-    border: 1px solid $ui-03;
+    background-color: $background-inverse;
```

Comment:
`$background-inverse` pairs with `$inverse-02` not `-01`. So this should either be `$icon-inverse` or `$text-inverse`